src/main/java/com/marklogic/kafka/connect/source/RowBatcherSourceTask.java

```diff
@@ -54,63 +48,54 @@ public final void start(Map<String, String> config) {
         logger.info("Starting RowBatcherSourceTask");
         Map<String, Object> parsedConfig = MarkLogicSourceConfig.CONFIG_DEF.parse(config);
         DatabaseClientConfig databaseClientConfig = new DefaultDatabaseClientConfigBuilder().buildDatabaseClientConfig(parsedConfig);
-        this.databaseClient = new DefaultConfiguredDatabaseClientFactory().newDatabaseClient(databaseClientConfig);
+        databaseClient = new DefaultConfiguredDatabaseClientFactory().newDatabaseClient(databaseClientConfig);
         dataMovementManager = databaseClient.newDataMovementManager();
         pollDelayMs = (Long) parsedConfig.get(MarkLogicSourceConfig.WAIT_TIME);
-        this.constraintColumn = (String) parsedConfig.get(MarkLogicSourceConfig.CONSTRAINT_COLUMN_NAME);
-        if (!StringUtils.hasText(constraintColumn)) {
-            constraintColumn = null;
-        }
 
-        queryContextBuilder = AbstractRowBatchBuilder.newQueryContextBuilder(dataMovementManager, parsedConfig);
+        queryHandler = QueryHandler.newQueryHandler(databaseClient, parsedConfig);
+        rowBatcherBuilder = AbstractRowBatchBuilder.newRowBatcherBuilder(dataMovementManager, parsedConfig);
         logger.info("Started RowBatcherSourceTask");
     }
 
     @Override
     public List<SourceRecord> poll() throws InterruptedException {
         // Temporary logging while testing CP to ensure the correct output format is being used
-        logger.info("Polling; QueryContextBuilder: " + queryContextBuilder.getClass().getName());
+        logger.info("Polling ; RowBatcherBuilder: " + rowBatcherBuilder.getClass().getName());
```

Comment:
Just do one log event here; I keep thinking to my previous job with using Loki + Grafana to look at log events, and having one logical log event spread across multiple lines was a real pain. There's only one log event here - logging the fact that we're polling along with a couple details - so we only want one log message.